fmf.spec

```diff
@@ -70,10 +73,12 @@ install -pm 644 fmf.1* %{buildroot}%{_mandir}/man1
 %doc README.rst examples
 %license LICENSE
 
-%files -n python%{python3_pkgversion}-%{name}
-%{python3_sitelib}/%{name}/
-%{python3_sitelib}/%{name}-*.egg-info
+%files -n python3-fmf -f %{pyproject_files}
```

Comment:
Why not just do `%py_provides python3-fmf` as is done in tmt?

Reply:
Can you expand on this? I am not sure what you mean here.

Reply:
So it's the same as was done in tmt, which was shipping separate lib and cli packages. It was deemed unnecessary and against the latest Fedora packaging guidelines, which are striving towards harmonizing rpms with Python packaging. fmf is also not building separate wheels for libs and for cli, and so should be done in rpm. 
For compatibility, `%py_provides python3-fmf` is added, so existing packages depending on python3-fmf will continue to work, as fmf package will provide python3-fmf.

Reply:
> which was shipping separate lib and cli packages

Here you mean `tmt` and `python3-tmt`? I partially agree, but in this case it don't think it make sense to have a `fmf` only package since the package is very much python centric with little OS-level dependencies other than `git`. I would say instead to have this as `python3-fmf` package only. For `tmt` it makes sense to be the opposite because of the `provision`.

`%py_provides` should be automatically handled when you install the site-packages file. I am not sure if `-f %{pyproject_files}` does that specifically. Will need to read/research about that.